src/Styleguide/Components/Lightbox/Lightbox.tsx

```diff
@@ -0,0 +1,286 @@
+import { color, space } from "@artsy/palette"
+import FadeTransition from "Components/Animation/FadeTransition"
+import { bind, once, throttle } from "lodash"
+import React from "react"
+import ReactDOM from "react-dom"
+import styled from "styled-components"
+import { Box } from "Styleguide/Elements/Box"
+import { Flex } from "Styleguide/Elements/Flex"
+import { CloseButton } from "./CloseButton"
+import { Slider, SliderProps } from "./LightboxSlider"
+
+const KEYBOARD_EVENT = "keyup"
+const ZOOM_PER_CLICK = 1.4
+const HIDE_ZOOM_SLIDER_AFTER = 2500
+
+const DeepZoomContainer = styled.div`
+  position: fixed !important;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 1000;
+  background-color: ${color("black100")};
+`
+
+export interface LightboxProps {
+  deepZoom: {
+    Image: {
+      xmlns: string
+      Url: string
+      Format: string
+      Overlap: number
+      TileSize: number
+      Size: {
+        Width: number
+        Height: number
+      }
+    }
+  }
+}
+
+export interface LightboxState {
+  shown: boolean
+  element: Element
+  viewer: any
+  deepZoomRef: any
+  slider: SliderProps
+  showZoomSlider: boolean
+  promisedDragon: Promise<any>
+  activityTimer?: NodeJS.Timer
+}
+
+export class Lightbox extends React.Component<LightboxProps, LightboxState> {
+  state = {
+    element: null,
+    viewer: null,
+    shown: false,
+    activityTimer: null,
+    showZoomSlider: true,
+    deepZoomRef: React.createRef(),
+    slider: {
+      min: 0,
+      max: 1,
+      step: 0.001,
+      value: 0,
+    },
+    /**
+     * FIXME: convert to import('openseadragon) once force supports it
+     */
+    promisedDragon: Promise.resolve(require("openseadragon")),
```

Comment:
Nice and fancy 👌 Is this lib particularly large?

Reply:
This doesn't actually do anything. It just mimics the API surface area of how it should be loaded. I believe that force would handle the async load okay, but other consumers of reaction might not...

open-sea-dragon is 200kb minified, 48kb gzipped. The maximum recommended size of an entry chunk is 250kb.

Reply:
Oh I know, I meant the forward compatibility to dynamic imports.

Gotcha.